pydantic_redis/model.py

```diff
@@ -1,17 +1,23 @@
 """Module containing the model classes"""
 import typing
-from typing import Optional, List, Any, Union, Dict
+from typing import Optional, List, Any, Set, Tuple, Union, Dict
 
 from redis.client import Pipeline
 
 from pydantic_redis.abstract import _AbstractModel
 
+NESTED_MODEL_PREFIX = "__"
```

Comment:
i think it is a good thing to centralize any kind of prefix.

Also I prefer `__%&l_` instead of your suggestion `___`: more readable

Reply:
Yeap...centralizing sounds like a good idea. Let's do it!

Reply:
Sure ...any prefix as long as it does not conflict with any other.